dockerfiles/centos/mi-dashboard/Dockerfile

```diff
@@ -22,26 +22,26 @@ FROM centos:7
 ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
 
 # install JDK Dependencies
-RUN yum install -y tzdata openssl curl ca-certificates fontconfig gzip tar \
+RUN yum install -y tzdata openssl curl ca-certificates fontconfig gzip tar nc unzip wget \
```

Comment:
Let's remove unzip and wget after build stage

Reply:
This is fixed with https://github.com/wso2/docker-ei/compare/91e605105f271e7940afecb5593bd4ea930e45cb..a01e1215316e43f60ed3879ecc896fe56abd9c87